test/manipulation.test.js

```diff
@@ -659,10 +659,60 @@ describe('manipulation', function() {
   });
 
   describe('updateOrCreate', function() {
+    var db = getSchema();
+    var Todo;
+
+    before(function setUp(done) {
+      Todo = db.define('Todo', {
+        content: String,
+      });
+      db.automigrate('Todo', done);
+    });
+    afterEach(function tearDown(done) {
+      db.automigrate(done);
```

Comment:
+1 on no point in doing this. Automigrate does not clear tables/collections (which BTW @bajtos we really need to optimize tests that need it. Much easier to TRUNCATE/DROP when needed -- not sure if this is more expensive, but I found myself needing it in many cases before). We should create a new feature request for these two commands that *actually* remove/clear the tables.